#1: added sample of VALUES array

tmtanzeel opened the pull request.

blackduck007 committed:
#1: added sample of VALUES array

sherubthakur commented on README.md:
````diff
@@ -36,8 +36,25 @@ Property	                  |	Type   	     |	Default	                      |	Desc
 ```
+import HorizontalTimeline from 'react-horizontal-timeline';
+
 const VALUES = [ /* The date strings go here */ ];
````
Let's remove this!

tmtanzeel replied:
this import is required to fix the compiler error:
`Failed to compile.

src/components/EducationTimeline/EducationTimeline.js
  Line 40:22:  'HorizontalTimeline' is not defined  react/jsx-no-undef

Search for the keywords to learn more about each error.
`

sherubthakur replied:
I was talking about removal of line `const VALUES = [ /* The date strings go here */ ];` and not the `import` one.

tmtanzeel replied:
removed (my apologies for the confusion)

sherubthakur commented on README.md:
```diff
@@ -36,8 +36,25 @@ Property	                  |	Type   	     |	Default	                      |	Desc
+/*
+Format: YYYY-MM-DD
+For example:
+const VALUES = [
+    '2008-06-01',
+    '2010-06-01',
+    '2013-06-01',
+    '2015-03-01',
+    '2019-01-01',
+    '2019-06-17',
+    '2019-08-01',
+];
```
No need for this section to be a comment and this can be part of the showcase code.

tmtanzeel replied:
addressed

nixypanda reviewed (commented):
Overall I like this idea. Just mentioned a few changes, once those are done we can totally merge this.

blackduck007 committed:
#1: addressed review comments given by actual author

tmtanzeel commented:
Subject: Addressed review comments
Please revisit the PR (
https://github.com/sherubthakur/react-horizontal-timeline/pull/94)

Thanks and regards
Tanzeel Mirza
Visit website <http://www.tanzeel.online>


On Sun, Sep 26, 2021 at 5:13 PM Sherub Thakur ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
>
> Overall I like this idea. Just mentioned a few changes, once those are
> done we can totally merge this.
> ------------------------------
>
> In README.md
> <https://github.com/sherubthakur/react-horizontal-timeline/pull/94#discussion_r716191104>
> :
>
> > @@ -36,8 +36,25 @@ Property	                  |	Type   	     |	Default	                      |	Desc
>  This is how it can be used.
>
>  ```
> +import HorizontalTimeline from 'react-horizontal-timeline';
> +
>  const VALUES = [ /* The date strings go here */ ];
>
> Let's remove this!
> ------------------------------
>
> In README.md
> <https://github.com/sherubthakur/react-horizontal-timeline/pull/94#discussion_r716191235>
> :
>
> > +const VALUES = [
> +    '2008-06-01',
> +    '2010-06-01',
> +    '2013-06-01',
> +    '2015-03-01',
> +    '2019-01-01',
> +    '2019-06-17',
> +    '2019-08-01',
> +];
>
> No need for this section to be a comment and this can be part of the
> showcase code.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sherubthakur/react-horizontal-timeline/pull/94#pullrequestreview-763687454>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH433VV7IBJP5RGWCHPV7HLUD4BP3ANCNFSM5EYPG47Q>
> .
>

tmtanzeel commented:
Subject: Addressed all review comments

Thanks and regards
Tanzeel Mirza
Visit website <http://www.tanzeel.online>


On Sun, Sep 26, 2021 at 7:32 PM Tanzeel Mirza ***@***.***> wrote:

> Subject: Addressed review comments
> Please revisit the PR (
> https://github.com/sherubthakur/react-horizontal-timeline/pull/94)
>
> Thanks and regards
> Tanzeel Mirza
> Visit website <http://www.tanzeel.online>
>
>
> On Sun, Sep 26, 2021 at 5:13 PM Sherub Thakur ***@***.***>
> wrote:
>
>> ***@***.**** commented on this pull request.
>>
>> Overall I like this idea. Just mentioned a few changes, once those are
>> done we can totally merge this.
>> ------------------------------
>>
>> In README.md
>> <https://github.com/sherubthakur/react-horizontal-timeline/pull/94#discussion_r716191104>
>> :
>>
>> > @@ -36,8 +36,25 @@ Property	                  |	Type   	     |	Default	                      |	Desc
>>  This is how it can be used.
>>
>>  ```
>> +import HorizontalTimeline from 'react-horizontal-timeline';
>> +
>>  const VALUES = [ /* The date strings go here */ ];
>>
>> Let's remove this!
>> ------------------------------
>>
>> In README.md
>> <https://github.com/sherubthakur/react-horizontal-timeline/pull/94#discussion_r716191235>
>> :
>>
>> > +const VALUES = [
>> +    '2008-06-01',
>> +    '2010-06-01',
>> +    '2013-06-01',
>> +    '2015-03-01',
>> +    '2019-01-01',
>> +    '2019-06-17',
>> +    '2019-08-01',
>> +];
>>
>> No need for this section to be a comment and this can be part of the
>> showcase code.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/sherubthakur/react-horizontal-timeline/pull/94#pullrequestreview-763687454>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AH433VV7IBJP5RGWCHPV7HLUD4BP3ANCNFSM5EYPG47Q>
>> .
>>
>

sherubthakur commented:
Looks good! Thanks for the PR :)
